### ISSUE TITLE
Semi-supervised learning

### ISSUE THREAD

**Issue by davidbenjamin** [davidbenjamin/permutect#42]
It's hard to label training data, especially with low read counts.  This is causing problems.  We can use semi-supervised learning to address this.  

The first approach to try is "Unsupervised Data Augmentation for Consistency Training" (https://arxiv.org/abs/1904.12848).  This employs an ordinary cross entropy loss on labeled data and a cross entropy loss between model predictions of unlabeled data and their augmented versions.  This is very simple to implement.
